src/features/customerList/customerListTableFilters/CustomerListTableFiltersList.tsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+import { Tag, IconCrossCircle } from 'hds-react';
+import { useTranslation } from 'react-i18next';
+import { useQuery } from '@apollo/react-hooks';
+import format from 'date-fns/format';
+
+import { FILTER_BOAT_TYPE_LABELS } from '../__generated__/FILTER_BOAT_TYPE_LABELS';
+import {
+  FILTER_PIER_LABEL,
+  FILTER_PIER_LABELVariables as FILTER_PIER_LABEL_VAR,
+} from '../__generated__/FILTER_PIER_LABEL';
+import {
+  FILTER_BERTH_LABEL,
+  FILTER_BERTH_LABELVariables as FILTER_BERTH_LABEL_VAR,
+} from '../__generated__/FILTER_BERTH_LABEL';
+import {
+  FILTER_WINTER_STORAGE_PLACE_LABEL,
+  FILTER_WINTER_STORAGE_PLACE_LABELVariables as FILTER_WINTER_STORAGE_PLACE_LABEL_VAR,
+} from '../__generated__/FILTER_WINTER_STORAGE_PLACE_LABEL';
+import {
+  FILTER_BOAT_TYPE_LABELS_QUERY,
+  FILTER_PIER_LABEL_QUERY,
+  FILTER_BERTH_LABEL_QUERY,
+  FILTER_WINTER_STORAGE_PLACE_LABEL_QUERY,
+} from '../queries';
+import { CustomerListTableFilters, FilterEntry, FilterEntryValue, ValueOf } from './types';
+import { createInterval, createDate } from './utils';
+import useListTableFilters, { getFiltersAsEntries, getEntriesAsFilters } from './useListTableFilters';
+import CustomerListTableFiltersListHarborLabel from './labels/CustomerListTableFiltersListHarborLabel';
+import CustomerListTableFiltersListWinterStorageAreaLabel from './labels/CustomerListTableFiltersListWinterStorageAreaLabel';
+import styles from './customerListTableFiltersList.module.scss';
+import { NOT_FOUND_STRING } from './constants';
+
+export interface Props {
+  filters: CustomerListTableFilters;
+}
+
+const CustomerListTableFiltersList = ({ filters }: Props) => {
+  const { t } = useTranslation();
+  const [, setFilters] = useListTableFilters();
+
+  const filtersAsEntries = React.useMemo(() => getFiltersAsEntries(filters), [filters]);
+
+  const boatTypesQuery = useQuery<FILTER_BOAT_TYPE_LABELS>(FILTER_BOAT_TYPE_LABELS_QUERY, {
+    skip: arrayFilterIsEmpty(filters.boatTypeIds),
+  });
+  const pierQuery = useQuery<FILTER_PIER_LABEL, FILTER_PIER_LABEL_VAR>(FILTER_PIER_LABEL_QUERY, {
+    skip: !filters.pierId,
+    variables: {
+      pierId: filters.pierId ?? '',
+    },
+  });
+  const berthQuery = useQuery<FILTER_BERTH_LABEL, FILTER_BERTH_LABEL_VAR>(FILTER_BERTH_LABEL_QUERY, {
+    skip: !filters.berthId,
+    variables: {
+      berthId: filters.berthId ?? '',
+    },
+  });
+  const winterStoragePlaceQuery = useQuery<FILTER_WINTER_STORAGE_PLACE_LABEL, FILTER_WINTER_STORAGE_PLACE_LABEL_VAR>(
+    FILTER_WINTER_STORAGE_PLACE_LABEL_QUERY,
+    {
+      skip: !filters.winterStoragePlaceId,
+      variables: {
+        winterStoragePlaceId: filters.winterStoragePlaceId ?? '',
+      },
+    }
+  );
```

Comment:
These could still use cache first fetch policy to avoid repeated requestt when opening and closing filters